client/src/js/indexes/components/Item.js

```diff
@@ -39,6 +40,7 @@ const IndexItemTop = styled.h3`
 
     > div {
         flex: 1 0 auto;
+        max-width: 70px;
```

Comment:
![image](https://user-images.githubusercontent.com/5943558/111389794-cedc4a00-866e-11eb-9dfc-5520cd6e7802.png)

I believe this line is causing this. If you are trying to style `IndexItemIcon`, modify the styles for `StyledIndexItemIcon`.